### PR TITLE
Sketch Element

### PR DESCRIPTION
**Pull request by thzanger**
The page "sketch.rst" was rewritten. Furthermore, the page "redlining.rst" was deleted and its contents merged into sketch.rst where appropriate.

### CODE REVIEW

File: en/functions/editing/sketch.rst [hunk 1]
```diff
@@ -3,36 +3,100 @@
...
+* Text
 
-.. image:: ../../../figures/sketch.png
-     :scale: 80
 
 Configuration
-================
+=============
+
+.. image:: ../../../figures/de/sketch_configuration.png
```
Comment (rschwammborn) [inline en/functions/editing/sketch.rst:21]: The PR also removes this image.

File: en/functions/editing/sketch.rst [hunk 2]
```diff
@@ -3,36 +3,100 @@
...
+* **Title:** Title of the elements. This is referenced by the button which opens the element.
+* **Target:** ID of the map-element, to which the Redlining refers.
+* **Geometrytypes** Lists all geometrytypes
+
+Draw geometries
+---------------
+
+Point drawing can be activated/deactivated via click on the button "point". Points are set via a single click on the map.
+
+.. image:: ../../../figures/de/sketch_create_points.png
```
Comment (rschwammborn) [inline en/functions/editing/sketch.rst:35]: I guess you referenced to the wrong screenshots (German files), also see links below.

File: en/functions/editing/sketch.rst [hunk 3]
```diff
@@ -3,36 +3,100 @@
...
+Drawn sketches can be individually deleted [1] and edited [2]. There is also the possibility to zoom to particular geometries [3].
+
+.. image:: ../../../figures/de/sketch_delete_edit_zoom.png
+     :scale: 80
+
+
+Stop drawing
+------------
+
+To stop drawing click the button "stop drawing".
```
Comment (rschwammborn) [inline en/functions/editing/sketch.rst:82]: better: pause button

**Comment by thzanger** [comment]
This PR will be closed, because it is a duplicate.

**Pull request closed without merging**